content/blog/2023-12-20-changepoint_explore.Rmd

```diff
@@ -0,0 +1,91 @@
+---
+title: Exploratory Analysis - Identifying Changing Variant Behavior during a Pandemic
+author: Tara Lakdawala, Ananya Joshi
+date: 2023-12-20
+tags:
+  - epidata
+  - changepoint analysis
+  - early indicators
+authors:
+  - tara
+  - ajoshi
+heroImage: changepoint-hero.png
+heroImageThumb: changepoint-blog.png
+
+summary: | 
+  We use changepoint detection algorithms to analyze Delphi's indicators and classify them as early, on-time,    late, undefined, or undetermined. 
+
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+
+## Introduction
+
+In November 2021, less than eighteen months after the first emergence of COVID-19, the National Institute for Communicable Diseases (NICD) in South Africa [announced a new COVID-19 variant](https://www.nicd.ac.za/covid-19-update-delta-variant-in-south-africa/). We subsequently discovered that the [transmission, hospitalization](https://www.yalemedicine.org/news/covid-19-variants-of-concern-omicron), and [mortality rates](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10125879/) of the virus variant, known as the Delta variant, differed from the Original pandemic strain^[We refer to the “Original” variant as both Alpha and Beta variants because they emerged [around the same time](https://www.verywellhealth.com/covid-variants-timeline-6741198#toc-b117-alpha).]. Methods that can detect these changes in underlying distributions of values in public health indicators (like cases, hospitalizations and deaths) can be useful in identifying changing health dynamics, like an emerging virus variant. We specifically use a technique called **changepoint detection** to analyze the power of Delphi’s historically available [indicators](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_signals.html) to identify changes in COVID-19 variant behavior between January 30, 2021 and September 17, 2022^[Range based on the data history available for many of the Delphi indicators]. 
```

Comment:
> We subsequently discovered

This sounds to me like we, at Delphi, discovered this- maybe change to "scientists"?

Reply:
Sounds good! Just changed it!